app/api.js

```diff
@@ -94,11 +99,28 @@ export function addFileFromFSPath(filePath) {
                    */
                   .then(res => resolve([...items, wrapper]))
               )
+            if (!Settings.getSync('skipGatewayQuery')) {
```

Comment:
* Fix the indentation
* refactor the `.then` chain in callback style
* query also the links (not just the wrapper hash, as it does not contain the data)